pkg/controllers/cloud/networkpolicy.go

```diff
@@ -292,6 +296,8 @@ type securityGroupImpl struct {
 	retryOp *securityGroupOperation
 	// true if retry operation is ongoing.
 	retryInProgress bool
+	// true if cloud operation is ongoing.
+	cloudOpInProgress bool
```

Comment:
This will be extended to track any on-going cloud operation right. (Even for addressGroup)

Reply:
no, just for AppliedToGroup.. Later we might add types for different operation based on new VMP design